bugzooka/integrations/slack_socket_listener.py

```
@@ -212,207 +158,263 @@ def _process_mention(self, event: Dict[str, Any]) -> None:
+            self.logger.warning("PR analysis failed: %s", analysis_result["message"])
 
-                # Send the result
-                message_content = analysis_result["message"]
+        try:
+            from bugzooka.integrations.inference_client import get_inference_client
+
+            client = get_inference_client()
+            extra["total_tokens"] = client.last_total_tokens
+            extra["tool_calls_count"] = client.last_tool_calls_count
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)